src/fu-util.c

```diff
@@ -1051,51 +1051,24 @@ fu_util_clear_offline (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_history_remove_all_with_state (history, FWUPD_UPDATE_STATE_PENDING, error);
 }
 
-static gboolean
-fu_util_verify_update_all (FuUtilPrivate *priv, GError **error)
-{
-	g_autoptr(GPtrArray) devs = NULL;
-
-	/* get devices from daemon */
-	devs = fwupd_client_get_devices (priv->client, NULL, error);
-	if (devs == NULL)
-		return FALSE;
-
-	/* get results */
-	for (guint i = 0; i < devs->len; i++) {
-		g_autoptr(GError) error_local = NULL;
-		FwupdDevice *dev = g_ptr_array_index (devs, i);
-		if (!fu_util_filter_device (priv, dev))
-			continue;
-		if (!fwupd_client_verify_update (priv->client,
-						 fwupd_device_get_id (dev),
-						 NULL,
-						 &error_local)) {
-			g_print ("%s\tFAILED: %s\n",
-				 fwupd_device_get_guid_default (dev),
-				 error_local->message);
-			continue;
-		}
-		g_print ("%s\t%s\n",
-			 fwupd_device_get_guid_default (dev),
-			 _("OK"));
-	}
-	return TRUE;
-}
-
 static gboolean
 fu_util_verify_update (FuUtilPrivate *priv, gchar **values, GError **error)
 {
 	g_autoptr(FwupdDevice) dev = NULL;
 
-	if (g_strv_length (values) == 0)
-		return fu_util_verify_update_all (priv, error);
-
+	priv->filter_include |= FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE;
```

Comment:
why _IMAGE? I think you should still be able to copy the checksums to the local sverify store if there is no metadata on the LVFS for this device.

Reply:
Ah I didn't realize that's a valid use case.  Yes, then this should go away.